### PR TITLE
RA-1177 added Travis badge to readme

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openmrs-distro-referenceapplication
 ===================================
 
-[![Sauce Test Status](https://saucelabs.com/buildstatus/rkorytkowski)](https://saucelabs.com/u/rkorytkowski)
+[![Build Status](https://travis-ci.org/AdamGrzybkowski/openmrs-distro-referenceapplication.svg?branch=RA-1177)](https://travis-ci.org/AdamGrzybkowski/openmrs-distro-referenceapplication) [![Sauce Test Status](https://saucelabs.com/buildstatus/rkorytkowski)](https://saucelabs.com/u/rkorytkowski)
```
Comment (rkorytkowski) [inline README.md]: Replace AdamGrzybkowski with openmrs :)

Comment (AdamGrzybkowski) [inline README.md]: Done :)

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 openmrs-distro-referenceapplication
 ===================================
 
-[![Sauce Test Status](https://saucelabs.com/buildstatus/rkorytkowski)](https://saucelabs.com/u/rkorytkowski)
+[![Build Status](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication.svg?branch=RA-1177)](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication) [![Sauce Test Status](https://saucelabs.com/buildstatus/rkorytkowski)](https://saucelabs.com/u/rkorytkowski)
```
Comment (rkorytkowski) [inline README.md]: branch=master :)

Comment (AdamGrzybkowski) [inline README.md]: heh sorry for this :) done

**Pull request merged**